system/View/View.php

```diff
@@ -402,35 +402,41 @@ public function extend(string $layout)
 	/**
 	 * Starts holds content for a section within the layout.
 	 *
-	 * @param string $name
+	 * @param string $name Section name
+	 *
+	 * @return void
 	 */
-	public function section(string $name)
+	public function section(string $name): void
```

Comment:
Modifying this would break extensions.
```suggestion
	public function section(string $name)
```